shodan/cli/host.py

```diff
@@ -93,6 +93,11 @@ def host_print_pretty(host, history=False):
 
         # Show optional ssl info
         if 'ssl' in banner:
+            if 'cert' in banner['ssl'] and banner['ssl']['cert']:
+                if 'issuer' in banner['ssl']['cert']:
+                    click.echo('\t|-- Cert Issuer: "{}"'.format(', '.join([*[f"{'='.join(map(str,v))}" for i,v in enumerate(list(banner['ssl']['cert']['issuer'].items()))]])))
```

Comment:
No worries, it was more the idea with an attempt at trying to address it. Please feel free address as you see fit.

Reply:
Was there a reason that you did used `enumerate`, another `join` with `map` and converted it to a `list`? I think it could be simplified with just:

```python
issuer = ', '.join(['{}={}'.format(key, value) for key, value in banner['ssl']['cert']['issuer'].items()])
click.echo('\t|-- Cert Issuer: {}'.format(issuer))
```

I'm asking to make sure I'm not missing some edge cases. I've tested it a bit and the above seems to work.

Reply:
To be honest, I don't remember. I think at the time I was just trying to keep it as similar to the rest of the code as possible. 

Your version looks good and much cleaner to me so I would definitely go with that too.